mne/viz/backends/renderer.py

```diff
@@ -59,17 +59,15 @@ def set_3d_backend(backend_name):
        +--------------------------------------+--------+---------+
        | 3D function:                         | mayavi | pyvista |
        +======================================+========+=========+
-       | :func:`plot_source_estimates`        | ✓      |         |
+       | :func:`plot_source_estimates`        | ✓      | -       |
```

Comment:
it's partially implemented already?

Reply:
Whoops! It's not on this PR but in [here](https://github.com/mne-tools/mne-python/pull/6232) with the `Brain` object, I'll remove it.